OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -456,26 +512,958 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <locus target="5r"/>
+                                    <dimensions type="leaf" unit="mm" xml:id="leafdim1">
+                                        <height>284</height>
+                                        <width>206</width>
+                                    </dimensions>
+                                    <note corresp="#leafdim1">Data on leaves dimensions taken from <locus target="#8r"/>.</note>
+                                </extent>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                            <layout columns="1" writtenLines="24">
+                                <locus from="1r" to="146v"/>
+                                <note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#8r"/>.</note>
+                                <dimensions unit="mm" xml:id="textarea">
+                                    <height>207</height>
+                                    <width>155</width>
+                                </dimensions>
+                                <dimensions type="margin" unit="mm" xml:id="margin">
+                                    <dim type="top">30</dim>
+                                    <dim type="bottom">50</dim>
+                                    <dim type="right">35</dim>
+                                    <dim type="left">15</dim>
+                                </dimensions>
+                                <note> The characters per line are 7–24.</note>
+                                <ab type="pricking">Ruling and pricking are visible. Both primary <!-- ??? --> and ruling pricks are visible.</ab>
+                                <ab type="ruling" subtype="pattern">Ruling pattern: 1-1-11:D!/0/0/C and 1-1-11:E!/0/0/C.</ab>
+                                <ab type="ruling">The vertical lines sometimes end around the pricks in the lower part of the folio, sometimes continue 
+                                    to the lower margin of the folio, and their
+                                    upper end is frequently difficult to trace. The lineation sometimes reaches slightly beyond the outer line of justification.
+                                </ab>
+                                <ab type="ruling">The upper line seems to be written above the ruling, although it is frequently difficult to
+                                    determine with certainty. <!-- ??? --> The bottom line is written above the ruling.
+                                </ab>
+                            </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <decoDesc>
+                            
+                            <decoNote xml:id="d1" type="miniature">
```

Comment:
I think there is still a trace of a branch conflict here. The miniatures are already described by Jacopo in the common part. Can their list be deleted here? Or should their description be moved from the common part to p1?

Reply:
They should be in p1. Is there a way to compare the text in the different parts? I'm not sure if I revised the captions here or not